requirements.txt

```diff
@@ -1 +1 @@
-django>=2.2,<4.0
+django>=2.2,<5
```

Comment:
Why do you limit the Django version altogether? To be honest, that's really annoying. I don't see a reason why the last release wouldn't be working for most people. Except they can't update to Django 4 because you're blocking it.

Reply:
@codingjoe good point, Johannes, thank you for the suggestion!
 I removed upper condition here.